clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -438,6 +438,11 @@
                         "S999"
                     ],
                     "uniqueItems": true
+                },
+                "cobol-lsp.enableNativeBuild": {
```

Comment:
The configuration parameter name looks confusing to me. Will enableNativeServer be a better name? Maybe @zeibura or @zimlu02 can add something here?

Reply:
Server type: Java / Native